src/sentry/search/utils.py

```diff
@@ -72,6 +72,27 @@ def parse_percentage(value):
     return value / 100
 
 
+def parse_numeric_value(value):
+    try:
+        value = float(value)
+    except ValueError:
+        pass
+    else:
+        return value
+
+    numeric_multiples = {"k": 10.0 ** 3, "m": 10.0 ** 6}
```

Comment:
let's add `b` to this list as well to match the [frontend formats](https://github.com/getsentry/sentry/blob/master/src/sentry/static/sentry/app/utils/formatters.tsx#L171)